### PR TITLE
Add rules for publishing release-1.15 branches

### LINKED ISSUE #77871: Update publishing-bot for release-1.15
The branch [`release-1.15`][branch] was just created.

Please update the [publishing-bot's configuration][config] to also publish this
new branch.

[branch]: https://github.com/kubernetes/kubernetes/tree/release-1.15
[config]: https://github.com/kubernetes/kubernetes/tree/master/staging/publishing

/cc @kubernetes/publishing-bot-reviewers
/sig release
/area release-eng
/milestone v1.15

### PR DESCRIPTION
**Pull request by nikhita**
Fixes https://github.com/kubernetes/kubernetes/issues/77871

1.15 branch will be cut tomorrow, so I figured it would be good to have the PR out early this time. :) 
Ref: https://github.com/kubernetes/sig-release/tree/master/releases/release-1.15#timeline

This PR also removes rules for publishing patch releases for 1.11.

/hold
Until:
1. the 1.15 branch is actually created
2. it's confirmed that it's ok to remove 1.11 rules

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nikhita** [comment]
/kind cleanup
/priority important-soon
/assign @sttts @dims 

/cc @Bubblemelon 
branch release manager

/cc @tpepper @foxish 
Can you confirm if it is ok to remove rules for 1.11? v1.11.10 has been released but it isn't clear to me from https://github.com/kubernetes/org/pull/664 if more patch releases are intended?

**Comment by nikhita** [comment]
Related https://github.com/kubernetes/publishing-bot/pull/178

**Comment by dims** [comment]
/approve
/lgtm

I notice the hold with the list of TODO(s). +1 to those items.

**Comment by hoegaarden** [comment]
Now that https://github.com/kubernetes/release/pull/676 is merged, tomorrow when @Bubblemelon is cutting the release, we should see an issue popping up somewhere. 🤞

**Review by Bubblemelon — APPROVED** [review]
Thanks! :smile:

**Comment by Bubblemelon** [comment]
cc: @idealhack 

Re: @hoegaarden 
YangLi will be cutting beta 0 on my behalf. :tada:

**Comment by idealhack** [comment]
Thank you @nikhita, @hoegaarden, and @Bubblemelon.

The beta release which will create the branch is tracking at https://github.com/kubernetes/sig-release/issues/624.

**Comment by tpepper** [comment]
I know of no reason we'd make yet another "last" 1.11 release, but defer to @foxish.

@imkin when does the 1.11 test infra go way?  Soon, or?

I'm definitely fine with all the 1.11 things going away.

**Comment by foxish** [comment]
We've already made a last 1.11 release. We can get rid of the test infra
for it now IMO.

**Comment by imkin** [comment]
@tpepper the 1.11 jobs would be taken out on May 16th 2019.

**Comment by idealhack** [comment]
the 1.15 branch was created: https://github.com/kubernetes/kubernetes/tree/release-1.15

**Comment by nikhita** [comment]
Thanks, Yang!

/hold cancel

**Pull request merged**
